application/common/models/User.php

```diff
@@ -1278,6 +1281,16 @@ public function getEmailAddress(): string
         return $this->email ?? $this->personal_email ?? '';
     }
 
+    /**
+     * Gets query for [[EmailLogs]].
+     *
+     * @return \yii\db\ActiveQuery
+     */
+    public function getEmailLogs()
+    {
+        return $this->hasMany(EmailLog::class, ['to_address' => 'email']);
```

Comment:
Will this work if you replace `'email'` with `$this->getEmailAddress()`?

Reply:
I find it weird that this isn't in a BASE model, and not in an extended model class.

Reply:
As we discussed on Meet, changing the migration will solve this.